cluster/gce/gci/configure-helper-common.sh

```diff
@@ -617,12 +617,10 @@ function create-master-auth {
     for extra_component in "${extra_components[@]}"; do
       local token="$(secure_random 32)"
       append_or_replace_prefixed_line "${known_tokens_csv}" "${token}," "system:${extra_component},uid:system:${extra_component}"
-      if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" == "true" ]]; then
-        if [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
-          create-kubeconfig "${extra_component}" "${token}" "${PROXY_RESERVED_IP}" "8888" "http"
-        else
-          create-kubeconfig "${extra_component}" "${token}" "localhost" "8080" "http"
-        fi
+      if [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
```

Comment:
again, this falls back to assumption that RP or TP always use insecure ports connection

Reply:
In this version, yes - this is what we currently have today. When we have secure port support, we will be modifying this path.

Reply:
if this is the case, we probably need to reset this flag to TRUE in the default config if the RP or TP flags is set --- which is in the previous PR. so we can keep the previous flow here.